src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,132 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.people;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        validUser = new User();
+        validUser.setAge(19);
+        validUser.setLogin("javalogin");
+        validUser.setPassword("password");
+    }
+
+    @Test
+    void nullUser_NotOK() {
+        assertThrows(NullPointerException.class, () -> {
+            registrationService.register(null);
+        });
+    }
+
+    @Test
+    void emptyUser_NotOk() {
+        validUser.setAge(0);
+        validUser.setLogin("");
+        validUser.setPassword("");
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(validUser);
+        });
+    }
```

Comment:
i think this test is redundant

Reply:
maybe. but let the mentor say something about it